src/util/resourceValidationUtils.js

```diff
@@ -0,0 +1,124 @@
+const { resolveSchema } = require('@projecttacoma/node-fhir-server-core');
```

Comment:
Let's add some logger debug bois to this file, maybe some infos as well. Would be nice to know what's happening along the way